docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+web:
+  build: .
+  command: mix phoenix.server
+  environment:
+    PORT: 80
+  volumes:
+  - .:/app
+  ports:
+  - "80:80"
```

Comment:
I just think you are better off if you don't re-use ports among your dev apps. We know gravity runs on 80, so I wouldn't expose 80 as the port for this container, which is what the `ports:` directive does.